ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml

```diff
@@ -277,6 +277,25 @@ StatusSectionLayout {
                 readonly property CommunityTokensStore communityTokensStore:
                     rootStore.communityTokensStore
 
+                function setFeesInfo(ethCurrency, fiatCurrency, errorCode) {
```

Comment:
Does it make sense to make it `d` global? I thinks we will need it also for  the `airdrop` flow.

Reply:
Yes, My next task is extending airdrops. If we need to extend the scope I will do this.